seedlet/assets/sass/blocks/utilities/_editor.scss

```diff
@@ -149,15 +149,6 @@
 [data-block] {
 	margin-top: var(--global--spacing-vertical);
 	margin-bottom: var(--global--spacing-vertical);
-
-	[data-block]:first-child {
```

Comment:
I can confirm that removing these lines solves the social icons + buttons alignment issue in the editor.